modules/ossm-smcp-prod.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-deploy-production.adoc
+
+[id="ossm-smcp-prod_{context}"]
+= Configuring your ServiceMeshControlPlane resource for production
+
+If you have installed a basic `ServiceMeshControlPlane` resource to test {ProductShortName}, you must configure it to production specifications before you deploy your service mesh in production.
+
+You cannot change the `metadata.name` field of an existing `ServiceMeshControlPlane` resource. For production deployments you _must_ customize the default template. 
+
+.Procedure
+
+. Configure Jaeger for production. 
+.. Edit the `ServiceMeshControlPlane` resource to use the `production` deployment strategy, by setting `spec.addons.jaeger.install.storage.type` to 'Elasticsearch' and specify additional configuration options under `install`. You can create and configure your Jaeger instance and set `spec.addons.jaeger.name` to the name of the Jaeger instance, for example `jaeger-production`.
++
+.Control Plane default Jaeger parameters (Elasticsearch)
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+spec:
+  version: v2.0
+  tracing:
+    sampling: 100
+    type: Jaeger
+  addons:
+    jaeger:
+      name: jaeger-production
+      install:
+        storage:
+          type: Elasticsearch
+        ingress:
+          enabled: true
+  runtime:
+    components:
+      tracing.jaeger.elasticsearch: # only supports resources and image name
+        container:
+          resources: {}
+----
++
+. Ensure your security certificates are production ready. Install security certificates from an external certificate authority. For more information, see Security.
```

Comment:
I'm reading this as you ensure your security certificates are production ready *by* installing security certificates from an external authority. If my reading is right, it might be clearer if you said something like, 

`Ensure your security certificates are production ready by installing security certificates from an external certificate authority.`

Reply:
Also, do you mean to embed a link in `Security.` It reads as though you want it to include a link for users to visit. I don't see one though.